README.md

```diff
@@ -286,6 +286,11 @@ The crates included as part of Tracing are:
 
 * [`tracing-log`]: Compatibility with the `log` crate (unstable).
 
+* [`tracing-opentelemetry`]: provides a layer that connects spans from multiple
```

Comment:
Does this description make sense? couple of changes subscriber -> layer, and some more context on what otel could be used for

Reply:
I think it seems good!

Reply:
👍